packages/react-scripts/config/webpack.config.js

```diff
@@ -129,10 +129,13 @@ module.exports = function (webpackEnv) {
         loader: require.resolve('postcss-loader'),
         options: {
           postcssOptions: {
+            // Necessary for external CSS imports to work
+            // https://github.com/facebook/create-react-app/issues/2677
+            ident: 'postcss',
```

Comment:
I think the `ident` property is not available in `postcss-loader` v6.1.1

Reply:
@mwanagosos it's the postcss options not the loader - I'll resolve, I could not find any reference in the postcss changelog, but if you still think this is an issue please reopen with a reference to a changelog or documentation